scripts/explore_screened_crossreg.R

```diff
@@ -0,0 +1,349 @@
+
+trn_filtered <- standardize_pairs(trn_filtered)
+
+# Add columns to simplify identifying bidirectional/unidirectional linking, make upset plot easier to build
+# Add column to identify non-EUCTR registry
+# This version of the table separates bidirectional and unidirectional trial pairs
+# ie. no pair marked as bidirectional will also be marked as unidirectional
+
+# in order to obtain a table with all bidirectional pairs also marked as unidirectional:
+# uncomment the line with `unidirectional = if_else((trn1inreg2 | trn2inreg1), TRUE, FALSE)`
+# comment out the line with `unidirectional = if_else((trn1inreg2 | trn2inreg1) & !bidirectional, TRUE, FALSE)`
+
+trn_filtered <- trn_filtered |>
+  mutate(
+    bidirectional = if_else(trn1inreg2 & trn2inreg1, TRUE, FALSE),
+    is_title_matched = if_else(is.na(is_title_matched), FALSE, is_title_matched),
+    non_euctr_registry = if_else(registry1 == "EudraCT", registry2, registry1) ,
+    unidirectional = if_else((trn1inreg2 | trn2inreg1) & !bidirectional, TRUE, FALSE)
+    # unidirectional = if_else((trn1inreg2 | trn2inreg1), TRUE, FALSE)
+  ) 
+
+# Filter for true cross-registrations
+#confirmed_crossreg_standardized <- manual_validation |>
+#  filter(is_true_crossreg) |>
+#  standardize_pairs() |>
+#  select(standardized_pair, is_true_crossreg)
+
+# Join true crossreg with larger table by standardized_pair
+#confirmed_crossreg <- confirmed_crossreg_standardized |>
+#  left_join(trn_manual_checks_standardized, by = "standardized_pair") |>
+#  mutate(is_true_crossreg = ifelse(standardized_pair == "2010-023688-16_NCT01326767", TRUE, is_true_crossreg))|> # Manually change is_true_crossreg back to TRUE for row "2010-023688-16_NCT01326767", not sure why it changes at all
+#  filter(is_true_crossreg)
+
+# Filter for trials linked by publication in any way
+#pub_crossregs <- confirmed_crossreg |>
+#  filter(at_least_one_pub) |>
+#  mutate(
+#    trn1_in_pub_abs = replace_na(trn1_in_pub_abs, FALSE),
+#    trn2_in_pub_abs = replace_na(trn2_in_pub_abs, FALSE),
+#    trn_in_pub_abs = trn1_in_pub_abs | trn2_in_pub_abs,
+    
+#    trn1_in_pub_si = replace_na(trn1_in_pub_si, FALSE),
+#    trn2_in_pub_si = replace_na(trn2_in_pub_si, FALSE),
+#    trn_in_pub_si = trn1_in_pub_si | trn2_in_pub_si,
+    
+#    trn1_in_pub_ft = replace_na(trn1_in_pub_ft, FALSE),
+#    trn2_in_pub_ft = replace_na(trn2_in_pub_ft, FALSE),
+#    trn_in_pub_ft = trn1_in_pub_ft | trn2_in_pub_ft
+#  )
+
+# Filter for trials linked by FT mention
+#pub_ft <- pub_crossregs |>
+#  filter(trn1_in_pub_ft | trn2_in_pub_ft)
+
+# Filter for trials linked by SI mention
+#pub_si <- pub_crossregs |>
+#  filter(trn1_in_pub_si | trn2_in_pub_si)
+
+# Filter for trials linked by abstract mention
+#pub_abs <- pub_crossregs |>
+#  filter(trn1_in_pub_abs | trn2_in_pub_abs)
+
+############################################################################
+# Upset plot for screened TRN pairs (with all categories, not just publication linkages)
```

Comment:
remove "(with all categories, not just publication linkages)" from the comment, as this is confusing to a naïve reader